sklearn/linear_model/_base.py

```diff
@@ -419,6 +412,27 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+    normalize : bool, default=False
```

Comment:
I'm not sure we should move this down since the order of the parameters hasn't changed (yet)?

Reply:
same as previous comment